packages/react-experiments/src/components/PersonaCoin/PersonaCoin.view.tsx

```diff
@@ -8,6 +8,7 @@ import { hideInitialsWhenImageIsLoaded } from './propHelpers';
 import PersonaCoinSize10 from './PersonaCoinSize10/PersonaCoinSize10';
 import { PersonaCoinInitials } from './PersonaCoinInitials/PersonaCoinInitials';
 import type { IPersonaCoinComponent, IPersonaCoinProps, IPersonaCoinSlots } from './PersonaCoin.types';
+import { PersonaSize } from '@fluentui/react';
```

Comment:
feel free to ignore

```suggestion
import { PersonaSize } from '@fluentui/react';
import type { IPersonaCoinComponent, IPersonaCoinProps, IPersonaCoinSlots } from './PersonaCoin.types';
```

Reply:
I'll leave it as-is since we don't have any rules about sorting imports, and changing it will delay this PR.